src/modules/transaction/hooks/queries/useTransactions.js

```diff
@@ -35,17 +47,49 @@ import { useCustomInfiniteQuery } from 'src/modules/common/hooks';
  * @returns the query object
  */
 
-export const useTransactions = ({ config: customConfig = {}, options } = {}) => {
-  const config = {
-    url: `/api/${API_VERSION}/transactions`,
-    method: 'get',
-    event: 'update.transactions',
-    ...customConfig,
-    params: { limit, ...(customConfig?.params || {}) },
-  };
-  return useCustomInfiniteQuery({
-    keys: [TRANSACTIONS],
+export const useTransactions = ({
+  keys = [TRANSACTIONS],
+  config: customConfig = {},
+  options,
+  getUpdate,
+} = {}) => {
+  const [hasUpdate, setHasUpdate] = useState(false);
+  const queryClient = useQueryClient();
+  const config = useTransactionsConfig(customConfig);
+
+  /* istanbul ignore next */
+  const transactionUpdate = useCallback(() => {
+    setHasUpdate(true);
+  }, []);
+
+  useEffect(() => {
+    if (getUpdate) {
+      /* istanbul ignore next */
+      client.socket.on('new.transactions', transactionUpdate);
+      client.socket.on('delete.transactions', transactionUpdate);
+    }
+    return () => {
+      client.socket.off('new.transactions', transactionUpdate);
+      client.socket.off('new.transactions', transactionUpdate);
```

Comment:
guys, here you missed to remove the socket for `delete.transactions`. You double removed it for `new.transactions` :)